test/xbgpu/test_recv.py

```diff
@@ -129,6 +131,21 @@ def gen_heaps(layout: Layout, data: ArrayLike, first_timestamp: int) -> Generato
 class TestStream:
     """Test the stream built by :func:`katgpucbf.recv.make_stream`."""
 
+    @pytest.fixture
+    async def sensors(self) -> SensorSet:
+        """Receiver sensors."""
+        # This is an async fixture because make_sensors requires a running event loop
+        return make_sensors(sensor_timeout=1e6)  # Large timeout so that it doesn't affect the test
+
+    @pytest.fixture
+    def time_converter(self) -> TimeConverter:
+        """Time converter.
+
+        This is a simple implementation that keeps ADC and Unix timestamps
+        closely related to make tests easily readable.
+        """
+        return TimeConverter(1.0, 1000.0)
```

Comment:
Do you think it would make sense to move this fixture to the top level to avoid duplication?

Reply:
Yeah I was thinking about that too, but wondered whether I was enabling my own internal spec-creep. Can I move it to `test/__init__.py` ? Or would you prefer it to be an actual `fixture`, with `scope=module`?